tests/test_unit.py

```diff
@@ -1,7 +1,10 @@
 """Temporary unit test."""
-from ifk-scb-compilations.scb import temp
+from ifk_scb_compilations.main import main
 
 
 def test_unit_temp():
     """Unit test of temp function."""
-    assert temp() is True
+    assert main() is None
+
+
+test_unit_temp()
```

Comment:
Don't think this is needed. `pytest` should find the test anyway.

Reply:
removed.